api/app/src/external/commonwell/link/shared.ts

```diff
@@ -0,0 +1,48 @@
+import { Person } from "@metriport/commonwell-sdk";
+import { apiUrl } from "../api";
+import {
+  Patient,
+  PatientData,
+  PatientExternalData,
+  PatientExternalDataEntry,
+} from "../../../models/medical/patient";
+import { PatientDataCommonwell } from "../patient-shared";
+import { MedicalDataSource } from "../..";
+
+export const createPatientLink = (personId: string, patientId: string): string => {
+  return `${apiUrl}/v1/person/${personId}/patientLink/${patientId}/`;
+};
+
+export const commonwellPersonLinks = (persons: Person[]): Person[] => {
+  const links: Person[] = [];
+
+  persons.forEach(person => {
+    if (person) links.push(person);
+  });
+
+  return links;
+};
+
+export type PatientWithCW = Omit<Patient, "data"> & {
+  data: Omit<PatientData, "externalData"> & {
+    externalData: Omit<PatientExternalData, "COMMONWELL"> & {
+      [MedicalDataSource.COMMONWELL]: PatientDataCommonwell;
+    };
+  };
+};
+
+export function patientWithCWData(
+  patient: Patient,
+  cwEntry: PatientExternalDataEntry
```

Comment:
I helped with this and now I realized mu suggestion is unsafe: there's nothing preventing callers of this function to send a `PatientExternalDataEntry` that's not CW related.

I created a ticket to review this, just flagging so its known: https://github.com/metriport/metriport-internal/issues/427